### ISSUE TITLE
Docker Control (ctrl4docker@hoffis-eck.de): Error adding applet to panel

### ISSUE THREAD

**Issue by andreevlex** [linuxmint/cinnamon-spices-applets#229]
Hello 
Trying to set applet ctrl4docker and glass.log these are the errors.
```log
error t=2017-03-18T08:20:59.831Z [Applet "ctrl4docker@hoffis-eck.de"]: Failed to evaluate 'main' function on applet: ctrl4docker@hoffis-eck.de/31
error t=2017-03-18T08:20:59.831Z Could not load applet ctrl4docker@hoffis-eck.de
```
`lsb_release -a`
Distributor ID:	LinuxMint
Description:	Linux Mint 18.1 Serena
Release:	18.1
Codename:	serena

**Comment by Odyseus** [comment]
Hello, @andreevlex.

Open the applet.js file and change all instances of **docker-ctrl@hoffis-eck.de** to **ctrl4docker@hoffis-eck.de** and see if that fixes it. It has the wrong UUID defined inside that file.

**Comment by andreevlex** [comment]
works

**Comment by Odyseus** [comment]
@andreevlex: I'm sure that the reviewers will ask you to edit your pull requests titles and commit titles to use the full UUID of the applet. You should change them to something as the following:

**Docker Control (ctrl4docker@hoffis-eck.de): Fixes #229 incorrect UUID**

**Docker Control (ctrl4docker@hoffis-eck.de): added russian translation**

**About the applet author:** You can find it inside the info.json file. For this applet in particular, it says that the author is **hoffis-eck.de**, but there isn't a GitHub user with that name (and there can't be). But there is a GitHub user called @hoffis.

@hoffis: Can you confirm that you are the author of the Cinnamon applet called Docker Control with UUID ctrl4docker@hoffis-eck.de?
